### PR TITLE
Update: Modify iscsi/nfs image url for disconnect env

### PR DESCRIPTION
**Pull request by duanwei33**
Did some test for iscsi/nfs pass.

### CODE REVIEW

File: features/step_definitions/helper_services.rb [hunk 1]
```diff
@@ -484,7 +484,7 @@
       res = host.exec_admin(*setup_commands)
       raise "iSCSI initiator setup commands error" unless res[:success]
     end
-    @result = admin.cli_exec(:create, n: _project.name, f: 'https://raw.githubusercontent.com/openshift-qe/docker-iscsi/master/iscsi-target.json')
+    @result = admin.cli_exec(:create, n: _project.name, f: "#{ENV['BUSHSLICER_HOME']}/testdata/storage/iscsi/iscsi-target.json")
```
Comment (liangxia) [inline features/step_definitions/helper_services.rb:487]: We can use `BushSlicer::HOME` to replace `ENV['BUSHSLICER_HOME']`

Comment (duanwei33) [reply features/step_definitions/helper_services.rb:487]: Sure, let's follow the professional guidance :) 
Yes, I checked the other file, BushSlicer::HOME is more used, I updated the another one in this file.

File: features/step_definitions/helper_services.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
   step %Q/SCC "privileged" is added to the "system:serviceaccounts:<%= project.name %>" group/
   step %Q{I run the :create client command with:}, table(%{
-    | f | #{ENV['BUSHSLICER_HOME']}/testdata/storage/nfs/nfs-server.yaml |
+    | f | <%= BushSlicer::HOME %>/testdata/storage/nfs/nfs-server.yaml |
```
Comment (akostadinov) [inline features/step_definitions/helper_services.rb:13]: @liangxia , actually to make sure that whenever we decide to keep test data, we can quickly update all places, even if it is in a separate repo again, it is much better to do like 
```
  step 'I obtain test data file "testdata/storage/nfs/nfs-server.yaml"'
  step %Q{I run the :create client command with:}, table(%{
    | f | <%= BushSlicer::HOME %>/testdata/storage/nfs/nfs-server.yaml |
  })

**Comment by duanwei33** [comment]
@liangxia @akostadinov PTAL

**Pull request merged**